cli/src/build/onboarding/android/ui/app.tsx

```
@@ -19,19 +19,93 @@ import { copyFile, readFile } from 'node:fs/promises'
 import { homedir } from 'node:os'
 import { join, resolve as resolvePath } from 'node:path'
 import process from 'node:process'
-import { Alert, ProgressBar, Select } from '@inkjs/ui'
-import { Box, Newline, Text, useApp, useInput, useStdout } from 'ink'
+import { ProgressBar, Select } from '@inkjs/ui'
```

ESLint

[error] 'Select' is defined but never used.

(unused-imports/no-unused-imports)